### PR TITLE
I want to choose a channel in which i am creating an order

### PR DESCRIPTION
**Pull request by bartoszpietrzak1994**
screen shot 2018-08-29 at 10 01 32

### CODE REVIEW

File: src/Form/Type/NewOrderCustomerCreateType.php [hunk 1]
```diff
@@ -12,9 +13,14 @@
...
     {
-        $builder->add('customer', TextType::class, [
-            'label' => 'sylius_admin_order_creation.ui.new_customer_email',
-        ]);
+        $builder
+            ->add('customer', TextType::class, [
+                'label' => false,
+            ])
+            ->add('channel', ChannelChoiceType::class, [
+                'required' => true,
```
Comment (Zales0123) [inline src/Form/Type/NewOrderCustomerCreateType.php]: `true` is a default value, I suppose

File: src/Resources/config/app/ajax.yml [hunk 2]
```diff
@@ -49,8 +49,8 @@ sylius_admin_order_creation_ajax_product_variant_by_codes:
                 arguments: $code
 
 sylius_admin_order_creation_ajax_provide_available_shipping_methods:
-    path: /admin/orders/available-shipping-methods/{customerEmail}/{shipmentNumber}
-    methods: [POST]
+    path: /admin/orders/available-shipping-methods/{customerEmail}/{channel}/{shipmentNumber}
```
Comment (Zales0123) [inline src/Resources/config/app/ajax.yml]: I'd vote for `channelCode`

**Comment by Zales0123** [comment]
And I know that I promised I won't pick on UI, but we could remove these "Channel" headers, put both forms in separate [segments](https://semantic-ui.com/elements/segment.html) and leave only the "OR" divider, it would look much better imo :) 🐃

File: composer.json [hunk 3]
```diff
@@ -21,7 +21,8 @@
         "php": "^7.2",
 
         "sylius/sylius": "~1.3.0",
-        "friendsofsymfony/jsrouting-bundle": "^2.2"
+        "friendsofsymfony/jsrouting-bundle": "^2.2",
+        "friends-of-behat/page-object-extension": "^0.2.0"
```
Comment (GSadee) [inline composer.json]: Shouldn't it be in `require-dev` section?

**Comment by Zales0123** [comment]
Nice, thank you, Bartosz! 🎉

**Pull request merged**